packages/firestore/test/unit/remote/remote_event.test.ts

```diff
@@ -459,6 +459,9 @@ describe('RemoteEvent', () => {
     event = aggregator.createRemoteEvent(version(3));
     expect(event.documentUpdates.size).to.equal(0);
     expect(event.targetMismatches.size).to.equal(1);
+    expect(event.targetMismatches.get(1)).to.equal(
```

Comment:
Does it make sense to add test cases here that verify the new target purpose as well? If yes, this may be a complicated test. If that is the case, feel free to log a task in taskflow and leave a TODO(b/NNNNNN) comment here as a reminder.